lib/livebook/web_socket/server.ex

```diff
@@ -0,0 +1,210 @@
+defmodule Livebook.WebSocket.Server do
```

Comment:
It is more like an async client, than a websocket server, no? Where do/will we use it?